src/locales/cs.ts

```diff
@@ -204,141 +204,140 @@ export default {
+    our_lady_of_knock: 'Panny Marie z Knock',
+    our_lady_of_lanka: 'Panny Marie z Lanky',
+    our_lady_of_lebanon: 'Panny Marie z Lebanonu',
     our_lady_of_loreto: 'Panny Marie Loretánské',
     our_lady_of_lourdes: 'Panny Marie Lurdské',
-  //   our_lady_of_lujan_patroness_of_argentina: 'Our Lady of Luján, Patroness of Argentina',
-  //   our_lady_of_madhu: 'Our Lady of Madhu',
-  //   our_lady_of_mercy: 'Our Lady of Mercy',
+    our_lady_of_lujan_patroness_of_argentina: 'Panny Marie Lujánské, patronky Argentiny',
+    our_lady_of_madhu: 'Panny Marie z Madhu',
+    our_lady_of_mercy: 'Panny Marie milosrdenství',
     our_lady_of_mount_carmel: 'Panny Marie Karmelské',
     our_lady_of_mount_carmel_mother_and_queen_of_chile: 'Panny Marie Karmelské, Matky a Královny Čile',
-  //   our_lady_of_peace: 'Our Lady of Peace',
-  //   our_lady_of_perpetual_help: 'Our Lady of Perpetual Help',
-  //   our_lady_of_refuge: 'Our Lady of Refuge',
+    our_lady_queen_of_peace: 'Panny Marie Královnej pokoje',
+    our_lady_of_perpetual_help: 'Panny Marie Matky ustavičné pomoci',
+    our_lady_of_refuge: 'Pannu Marii, Útočiště hříšníků',
     our_lady_of_sorrows: 'Panny Marie Bolestné',
-  //   our_lady_of_the_gate_of_dawn: 'Our Lady of the Gate of Dawn',
-  //   our_lady_of_the_pillar: 'Our Lady of the Pillar',
+    our_lady_of_the_pillar: 'Panny Marie na Sloupu',
     our_lady_of_the_rosary: 'Panny Marie Růžencové',
-  //   our_lady_of_the_valley: 'Our Lady of the Valley',
-  //   our_lady_of_vladimir: 'Our Lady of Vladimir',
-  //   our_lady_of_walsingham: 'Our Lady of Walsingham',
-  //   our_lady_queen_of_poland: 'Our Lady, Queen of Poland',
+    our_lady_of_the_valley: 'Panny Marie z Údolí',  // TODO: `z Valle` or `z Údolí` or `z Valle de Catamarca`; the latter is IMHO quite concreate, as there are two or three shrines in different places. I found two instances: [Slovak](https://sk.wikipedia.org/wiki/San_Fernando_del_Valle_de_Catamarca); [Czech](https://www.lovecpokladu.cz/artefakty/c/cirkev-a-uzite-umeni-novoveku/cirkevni-predmety/svatostky/panna-maria/nuestra-senora-del-valle-de-cata/); see also https://en.wikipedia.org/wiki/Virgen_del_Valle
+    our_lady_of_vladimir: 'Panny Marie Vladimírské',
+    our_lady_of_walsingham: 'Panny Marie Walsinghamské',
+    our_lady_queen_of_poland: 'Panny Marie, Královny Polska',
     our_lord_jesus_christ_the_eternal_high_priest: 'Ježíše Krista, nejvyššího a věčného kněze',
-  //   our_lord_of_miracles: 'Our Lord of Miracles',
+    our_lord_of_miracles: 'Pána zázraků',
     presentation_of_mary: 'Zasvěcení Panny Marie v Jeruzalémě',
     queenship_of_mary: 'Panny Marie Královny',
     adalbert_of_prague_bishop: 'Sv. Vojtěcha, biskupa a mučedníka',
-  //   aelred_of_rievaulx_abbot: 'Saint Aelred of Rievaulx, Abbot',
-  //   aengus_of_tallaght_bishop: 'Saint Aengus, Bishop and Abbot',
+    aelred_of_rievaulx_abbot: 'Sv. Aelreda z Rievaulx, opata',
+    aengus_of_tallaght_bishop: 'Sv. Aengusa, biskupa a opata',
```

Comment:
If nominative is `Aengus` (Latin or at least latinized name of 2nd declension), classical Czech genitive would be `Aenga`, younger people may prefer `Aenguse`.

Reply:
My bet would be the more classical form: `Sv. Aenga, biskupa a opata`